crates/bmp-pkt/src/lib.rs

```diff
@@ -51,6 +51,24 @@ pub enum BmpMessage {
     V3(BmpMessageValue),
 }
 
+impl BmpMessage {
+
+    /// Returns the BMP Version from the BMP Common Header
+    /// u8 because there is no IANA registry for BMP Versions
+    pub fn get_version(&self) -> u8 {
```

Comment:
why not returning BmpVersion?

Reply:
did not expect it to exist since i did not find a iana registry for it but i'm glad it exists here :)